segmentation/utils/synthetic_spatial_datagen.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+import xarray as xr
+import pandas as pd
+import skimage.measure
+import scipy
+import os
+from scipy.spatial.distance import cdist
+
+
+# Constants for random distance matrix generation
+AB_DIST_MEAN = 100
+AB_DIST_VAR = 1
+
+AC_DIST_MEAN = 20
+AC_DIST_VAR = 1
+
+
+# Constants for random centroid matrix generation
+A_CENTROID_FACTOR = 0.5
+B_CENTROID_FACTOR = 0.9
+C_CENTROID_FACTOR = 0.4
+
+A_CENTROID_COV = [[1, 0], [0, 1]]
+B_CENTROID_COV = [[1, 0], [0, 1]]
+C_CENTROID_COV = [[1, 0], [0, 1]]
```

Comment:
Since these are all default values of function arguments, they can just be declared in the function args.  For example, in `direct_init_dist_matrix`:

```
def direct_init_dist_matrix(num_A=100, num_B=100, num_C=100,
                            distr_AB={'mean' : 100, 'var' : 1}, 
                            dist_AC={'mean' : 20, 'var' : 1},
                            seed=None):
```

Reply:
Yeah definitely! Forgot to take those globals out, will absolutely fix.